# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../avifile/glibc-2.3.4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some invalid assumptions that fail since glibc-2.3.4.

  - Rene Rebe <rene@exactcode.de>

--- ./include/utils.h.orig	2002-09-24 09:19:22.000000000 +0200
+++ ./include/utils.h	2005-01-06 19:48:45.122345888 +0100
@@ -4,6 +4,9 @@
 #include "avm_default.h"
 #include "formats.h"
 
+#undef __THROW
+#define __THROW
+
 /**
  *  Utility functions.
  *    plain C functions are here
